testplan/testing/multitest/suite.py

```diff
@@ -840,3 +840,20 @@ def _xfail_test(test):
         return test
 
     return _xfail_test
+
+
+def timeout(seconds=None):
+    """
+    Decorator for non-testcase method in a test suite, mainly used for
```

Comment:
I guess that for a single testcase it might work, because it just add 'timeout' attribute to the function, but for parametrized testcases it certainly cannot work. So we don't need to let user know that they can do like this. only use `@testcase(timeout=10)`.